File: Programacion/UD5/ejercicios1 tema5/src/ejercicio9.java
import java.util.Random;

public class ejercicio9 {
    public static void main(String[] args) {

        int[] array = new int[10];
        Random random = new Random();


        for (int i = 0; i < array.length; i++){

            array[i] = random.nextInt(21) -10;
            System.out.print(array[i] + " ");


        }

        for (int i = 0; i < array.length; i++){
            for (int j = 0; j < array.length; j++){
                if (i != j) {

                    if (array[i] == array[j] ){
                        System.out.println("El " + array[i] + " se repite");

                    }

                }
            }

        }


    }
}

File: Programacion/UD7/EjerciciosKotlin1/src/Ejercicio9.kt
import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    println("Mete numeros a la lista (99 para parar)")
    var lista = mutableListOf<Int>()
    var num = sc.nextInt()
    while (num != 99){
        lista.add(num)
        num = sc.nextInt()
    }

    println(lista)
    devolverSinDuplicados(lista)

}

fun devolverSinDuplicados(lista: MutableList<Int>) {
    println(lista.distinct())
}